.github/workflows/release-please.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 20
+          node-version: 23
```

Comment:
I suggest this uses an LTS version? (i.e. 22)

Reply:
Saw this comment, not sure if still applies:

> I left the integration tests in CJS so that the behaviour of requiring yargs in a CJS environment can be tested -- this requires Node 23.